src/main/java/co/tryterra/terraclient/impl/UserImpl.java

```diff
@@ -18,30 +18,36 @@
 
 import co.tryterra.terraclient.api.User;
 import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
 import com.fasterxml.jackson.annotation.JsonProperty;
 import com.fasterxml.jackson.databind.JsonNode;
 
 import java.time.OffsetDateTime;
 import java.time.format.DateTimeFormatter;
 
+@JsonIgnoreProperties(ignoreUnknown = true)
```

Comment:
One of these annotations is redundant. You either need it on the class or on the creator - might want to double check which